src/components/Calendar/Calendar.jsx

```diff
@@ -35,6 +80,51 @@ const Calendar = ({
     setTimeout(() => setAnimationDirection(""), 300)
   }
 
+  const handleDateClick = (day, month, year) => {
+    const selectedDate = { day, month, year }
+
+    function convertDateObjectToString(dateObject) {
+      const { day, month, year } = dateObject
+      return `${String(month + 1).padStart(2, '0')}/${String(day).padStart(2, '0')}/${year}`
+    }
+    
+    const dateString = convertDateObjectToString({ day, month, year })
+
+    if (isSearchBarCalendar) {
+      if (!pickedCheckIn || (pickedCheckIn && pickedCheckOut)) {
```

Comment:
on airbnb once you selected the check-in and check-out dates you can still modify the selected time range by clicking on the different dates ( before the check-in date and/or after check-out date) and the date selection is adjusted accordingly. This feature is not mention is ACs for this PR so I am just pointing this out as optional.
<img width="781" alt="Screenshot 2024-11-11 at 22 08 58" src="https://github.com/user-attachments/assets/9342d038-fe94-48c2-82dc-328adc368c9d">
<img width="777" alt="Screenshot 2024-11-11 at 22 09 05" src="https://github.com/user-attachments/assets/f0422693-3c6b-4087-bfaf-1b03c1d2a44d">
<img width="845" alt="Screenshot 2024-11-11 at 22 09 14" src="https://github.com/user-attachments/assets/a1a65b38-f1f2-4e05-b15d-50f969f3a715">
<img width="791" alt="Screenshot 2024-11-11 at 22 09 21" src="https://github.com/user-attachments/assets/31c9201c-017b-4fd6-a392-95a71d49a44e">